db/migrations/2023080311350006-add-comments-github-to-comments-table.js

```diff
@@ -0,0 +1,165 @@
+const { Op } = require("sequelize");
+const Octokit = require("octokit").Octokit;
+const ChainModel = require("../models/chain.model");
+const NetworkModel = require("../models/network.model");
+const IssueModel = require("../models/issue.model");
+const TokenModel = require("../models/tokens.model");
+const RepositoryModel = require("../models/repositories.model");
+const PullRequestModel = require("../models/pullRequest.model");
+const MergeProposalModel = require("../models/mergeproposal");
+const CuratorsModel = require("../models/curator-model");
+const BenefactorModel = require("../models/benefactor.model");
+const DisputeModel = require("../models/dispute-model");
+const UserPaymentsModel = require("../models/user-payments");
+const DeveloperModel = require("../models/developer.model");
+const CommentsModel = require("../models/comments.model");
+const UserModel = require("../models/user");
+
+const { SKIP_MIGRATION_SEED_COMMENTS_DATE_GITHUB, NEXT_GH_TOKEN } = process.env;
+const BOT_NAME = 'bepro-bot'
+
+async function handleAddComments(comment, id, type, prId){
+  const regexGithubLogin = /@(\w+)/;
+  const bodyGithubLogin = regexGithubLogin.exec(comment.body);
+
+  const getCommentCreateData = (userId, userAddress, body) => ({
+    userId,
+    userAddress,
+    comment: body,
+    issueId: id,
+    hidden: false,
+    type,
+    created_at: comment.created_at,
+    updated_at: comment.updated_at,
+    ... prId ? { deliverableId: prId } : null
+  }) 
+
+  const getUser = (name) => UserModel.findOne({
+    where: {
+      githubLogin: name
+    }
+  })
+
+  if(bodyGithubLogin && comment.user.login === BOT_NAME){
+    const user = await getUser(bodyGithubLogin[1])
+    let text;
+
+    if(user){
+      //Example text: "@name  is working on this."
+      if(/working/.test(comment.body) && type === 'issue') text = `i'm working on this.`
+      //Example text: "has a solution - [check your bounty](https://)"
+      if(/solution/.test(comment.body) && type === 'issue') {
+        const regexSolution = /(?<=- ).*/
+        text = `finished a solution - ${comment.body.match(regexSolution)[0]}`
+      }
+      //Example text: "reviewed this Pull Request with the following message:    Message"
+      if(/Pull Request/.test(comment.body) && type === 'deliverable') {
+        const regexPr = /(?<=:\s*).*/;
+        text = comment.body.match(regexPr)[0]
+      }
+
+      await CommentsModel.create(getCommentCreateData(user.id, user.address, text))
+    }
+  } else {
+    const user = await getUser(comment.user.login)
```

Comment:
variable shares the same name as the variable responsible for this else clause (line 44); While this is possible, because `const` is scoped in its `{}` space, it makes for harder readability; Either make `user` be a `let user` or rename this variable.